src/routes/docs/users.js

```diff
@@ -329,4 +329,142 @@ usersDocs[`${apiPrefix}/resetProfileImg`] = {
   },
 };
 
+usersDocs[`${apiPrefix}/isBanned`] = {
+  get: {
+    tags: [tag],
+    summary: "본인의 현재 정지 기록을 가져움",
+    description:
+      "정지 기록들 중 본인이고, 서버 시간을 기준으로 expireAt 보다 작은 경우에 해당하는 정지 기록을 모두 가져옴",
+    responses: {
+      200: {
+        content: {
+          "application/json": {
+            schema: {
+              type: "array",
+              items: {
+                properties: {
+                  userId: {
+                    type: "string",
+                    description: "사용자의 UUID",
+                    example: "664b5c1bfd906295bc0639a9",
```

Comment:
objectId의 경우 해당 라인 추가하고 pattern을 통해 문서화를 진행하면 자동으로 example이 생성됩니다! 실제 objectId를 사용하는 것보다 해당 방법을 사용하는걸 권유드립니다.
`const { objectId } = require("../../modules/patterns"); `
```suggestion
                    pattern: objectId.source,
```

Reply:
오! 그런 방법이 있었군요! 알려주셔서 감사합니다!